.github/workflows/deploy.yml

```diff
@@ -0,0 +1,54 @@
+name: Deploy test version
+
+on:
+  pull_request:
+    types: [ labeled ]
```

Comment:
What label does the pull request need to have?

Reply:
Yes this allow to run the PR only on PR labeled "Deploy-test-version"